tests/modules/basics.c

```diff
@@ -417,7 +417,13 @@ int TestTrimString(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     size_t string_len = RedisModule_MallocSizeString(s);
     RedisModule_TrimStringAllocation(s);
     size_t len_after_trim = RedisModule_MallocSizeString(s);
-    if (len_after_trim < string_len) {
+#if defined(USE_JEMALLOC)
```

Comment:
i'm not certain we can rely on USE_JEMALLOC (modules built separately).
maybe use RedisModule_GetServerInfo?

Reply:
Done